vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -197,6 +197,12 @@ bool FrameworkElementViewManager::UpdateProperty(
         } else {
           xaml::Automation::AutomationProperties::SetAccessibilityView(element, winrt::AccessibilityView::Raw);
         }
+        if (auto control = nodeToUpdate->GetView().try_as<xaml::Controls::Control>()) {
+          if (control.IsTabStop() != propertyValue.AsBoolean()) {
+            control.IsTabStop(false);
+            xaml::Automation::AutomationProperties::SetAccessibilityView(element, winrt::AccessibilityView::Raw);
+          }
+        }
```

Comment:
Consider adding logic matching `ViewViewManager` to `ControlViewManager`, instead of this fallback in `FrameworkElementViewManager` It gets potentially confusing for multiple classes in the same hierarchy to mutate the same properties on the control. Reading `accessible` in those two view managers would fix the `focusable` implementation everywhere we provide `focusable` I think.